### DISCUSSION TITLE
Octodash having trouble connecting without octoprint remote desktop running

### DISCUSSION THREAD

**Discussion by tfaz099** [UnchartedBull/OctoDash#2221] [Q&A] [answered]
Raspi with Octoprint build always worked fine with octodash whether or not i was logged into the octoprint desktop interface on my mac.

I tried changing firmware on my printer to test something out, and when i switched back to my original firmware on printer (none of this should have anything to do with dash or octoprint) octodash gets stuck on that sleep menu... press to connect... then it says cannot connect, press screen to try again.

anyone have ideas on this ? is it a bug that i should report?

Note: when it does this and i open octoprint desktop on my mac... it loads all by itself from that "cannot connect" state

Another thing to be aware of is i did change the baudrate to 250000 to try it in firmware and octoprint console... but put it back to 112500...

thank you
tony

**Comment by UnchartedBull** [comment, marked as answer]
next time you connect via the OctoPrint webui check the "save connection settings" checkbox before clicking connect. This should fix your issue.
